pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -1001,6 +1002,12 @@ def create_model(df_sets, df_parameters, default={}):
         doc="Weekly processing (e.g. clarification) capacity per storage site [bbl/week]",
         mutable=True,
     )
+    model.p_sigma_NetworkNode = Param(
+        model.s_N,
+        default=float("inf"),
+        initialize=df_parameters["NodeCapacities"],
+        doc="Daily capacity per network node [bbl/week]",
```

Comment:
Similar to above, I think this should be "weekly capacity  per network node [bbl/week]". I will update it "daily" capacity in the unit conversion PR